<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Stefan Nikolaus &lt;stefan.nikolaus@kdemail.net&gt;</author>
 <comment>This file is part of the KDE project.
Copyright 2010 Stefan Nikolaus &lt;stefan.nikolaus@kdemail.net&gt;

This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 2 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program; if not, write to the Free Software
Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
</comment>
 <class>ConsolidateDetailsWidget</class>
 <widget class="QWidget" name="ConsolidateDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>162</width>
    <height>85</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="m_rowHeader">
     <property name="text">
      <string>Description in rows</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_columnHeader">
     <property name="text">
      <string>Description in columns</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_copyData">
     <property name="text">
      <string>Copy data</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
